pkg/pillar/hypervisor/kvm.go

```diff
@@ -102,6 +102,19 @@ const qemuConfTemplate = `# This file is automatically generated by domainmgr
 [realtime]
   mlock = "off"
 
+[drive]
```

Comment:
Wait, why are we changing this all of a sudden? What's wrong with the `ovmf.bin` that has worked for us for quite some time? @temerkhanov

Reply:
ovmf.bin apparently was hanging on startup on Supermicro HW that I have. This PR adds an ability to build a debug version and see what is going on in such occasions